### ISSUE TITLE
Option to trim code content (whitespaces, break lines)

### ISSUE THREAD

**Issue by atilacamurca** [znicholasbrown/vuepress-plugin-code-copy#12]
Can we have an option to trim the content?

This is the scenario:

1. The docs give generic instructions to, for example, run a bash command
1. If the command is just some kind of template, e.g: `create user my-username`
1. If you copy this with the plugin and paste on terminal, you don't have the chance to change `my-username` to anything else, because of the break line at the end.

**Comment by znicholasbrown** [comment]
Hi @atilacamurca, thanks for opening this; what do you propose for that as an option? Something like an optional config var `trim: true`?

**Comment by atilacamurca** [comment]
Yes, for backwards compatibility. The default value should `false`, and optionally you can change.

```js
module.exports = {
  plugins: [
    ["vuepress-plugin-code-copy", {
        trimContent: Boolean, // default: false
        }
    ]
  ]
}
```

PS. Do you have access to individual code snippets? Is there any way to pass options to individual code snippets? Something like <https://v1.vuepress.vuejs.org/guide/markdown.html#line-highlighting-in-code-blocks>, where you can pass props to the code component?
